linode_api4/linode_client.py

```diff
@@ -871,6 +871,11 @@ def ips(self, *filters):
     def ipv6_ranges(self, *filters):
         return self.client._get_and_filter(IPv6Range, *filters)
 
+    def ip_ranges_list(self):
+        result = self.client.post("/networking/ipv6/ranges", model=self)
```

Comment:
here the method should be `GET` https://www.linode.com/docs/api/networking/#ipv6-ranges-list